cobbler/autoinstallgen.py

```diff
@@ -52,6 +55,14 @@ def __init__(self, collection_mgr):
         self.templar = templar.Templar(collection_mgr)
 
     def createAutoYaSTScript(self, document, script, name):
+        """
+        This method attaches a script with a given name to an existing autoyast xml file.
+
+        :param document: The existing autoyast xml file.
```

Comment:
SUSE calls it "AutoYaST", right? ;-)

Consistency: either use "xml" or "XML" in capital letters everywhere.